app/models/cqm/measure.rb

```diff
@@ -79,6 +77,10 @@ class Measure
 
     scope :by_measure_id, ->(id) { where(measure_id: id) }
 
+    def all_stratifications
```

Comment:
this needs to be added to the mongoose schema as well

Reply:
done.